AMD-MAINTENANCE-GUIDE.md

```diff
@@ -0,0 +1,338 @@
+# Awesome-MakerDAO: Maintenance Guide
+
+Awesome-MakerDAO is a community-curated list of MakerDAO-related content and resources to which anyone can contribute. The long-term goal is developing a self-maintaining document, where anyone can get paid to add or update content and find and replace typos, with clear guidelines around how to update its resources.
+
+This guide details the suggested logic contributors should follow to improve Awesome-MakerDAO.
+
+- Follow the provided guidelines to help preserve a consistent and clean formatting style.
+- Keep Awesome-MakerDAO relevant. 
+  - Add, delete, and relocate resources when appropriate.
+  - Request admin approval before removing resources or when restructuring sections.
+
+**Bounties and Incentives**
+
+The MakerDAO Community Grants program supports members that provide useful contributions and improve Maker's document base. Inquire about bounties and incentives in the #Community-Development channel on MakerDAO's [Rocket.Chat](https://chat.makerdao.com).
```

Comment:
- Let not reference 'rocket.chat', it's a brand name. Let's stick with just 'our chat' or the 'MakerDAO chat'.
- linking directlly to rooms is easiest: `...incentives in the [#Community-Development channel](https://chat.makerdao.com/channel/community-development) on MakerDAO's chat.`

Reply:
noted